boilerplate/config/config.default.ts

```diff
@@ -1,6 +1,6 @@
 import { EggAppConfig, EggAppInfo, PowerPartial } from 'egg';
 
-export default (appInfo: EggAppInfo) => {
+export default (appInfo: EggAppInfo): PowerPartial<EggAppConfig> => {
```

Comment:
The return type can not be merged to EggAppConfig if you add this return type annotation

Reply:
Is there any interface that can be merge to EggAppConfig?
do you have any suggestion?

Reply:
return type of export function in `config.default.ts` will merge to EggAppConfig, sorry, I don't has any idea to fix this yet, So I think it's better to keep `"noImplicitAny": false` now.